src/runtime/components/nuxt-icon.vue

```diff
@@ -1,38 +1,81 @@
 <template>
-  <span class="nuxt-icon" :class="{ 'nuxt-icon--fill': !fill }" v-html="icons[name]" />
+  <div class="nuxt-icon-container">
```

Comment:
Don't use a container, if people want to use a icon, they expect to find a single element with the icon. The way it is set up now, you get a icon in a div in a div. If users would like to center their icon, they will do so themselves.